sound/soc/sof/ipc4-topology.c

```diff
@@ -1419,13 +1419,55 @@ static int sof_ipc4_control_load_volume(struct snd_sof_dev *sdev, struct snd_sof
 	return 0;
 }
 
+static int sof_ipc4_control_load_bytes(struct snd_sof_dev *sdev, struct snd_sof_control *scontrol)
+{
+	struct sof_ipc4_control_data *control_data;
+	int ret;
+
+	scontrol->size = sizeof(struct sof_ipc4_control_data) + scontrol->priv_size;
```

Comment:
don't u need checks for max size? look at IPC3 for exmaple

Reply:
Will update the patch. I did add the max size support before, but not apply those change into this patch.